src/Tribe/Editor/Blocks/Event_Datetime.php

```diff
@@ -98,7 +100,7 @@ public function block_data() {
 				],
 				'timeZoneLabel' => [
 					'type'    => 'string',
-					'default' => class_exists( 'Tribe__Timezones' ) ? Tribe__Timezones::wp_timezone_string() : get_option( 'timezone_string', 'UTC' ),
+					'default' => class_exists( 'Tribe__Timezones' ) ? Tribe__Events__Timezones::get_event_timezone_abbr( $event_id ) : get_option( 'timezone_string', 'UTC' ),
```

Comment:
This `class_exists()` isn't looking for the correct class anymore, since we're using a different class from what we're checking.